code/src/index.css

```diff
@@ -1,13 +1,199 @@
+*{
+  margin: 0px;
+  padding: 0px;
+}
+
 body {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: black;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.body{
+  position: relative;
+  padding: 10px 30px 10px 30px;
+  margin: 10px auto;
+}
+
+header{
+  color: white;
+  font-size: 15px;
+  padding: 10px 0px;
+  margin: 50px 0px 10px 45px;
```

Comment:
Header seem to align to the right in the tablet and mobile. Maybe try to remove the margin to the left?